packages/devtools_app/test/resolved_uri_manager_test.dart

```diff
@@ -0,0 +1,155 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: import_of_legacy_library_into_null_safe
+
+import 'package:devtools_app/src/service/resolved_uri_manager.dart';
+import 'package:devtools_app/src/service/service_manager.dart';
+import 'package:devtools_app/src/shared/globals.dart';
+import 'package:devtools_test/devtools_test.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import 'package:vm_service/vm_service.dart';
+
+void main() async {
+  group('ResolvedUriManager', () {
+    const String isolateId = 'anIsolateId';
+    const uri1 = 'this/is/a/uri1';
+    const uri2 = 'this/is/a/uri2';
+    const packageUri1 = 'uri/am/i1';
+    const packageUri2 = 'uri/am/i2';
+    ResolvedUriManager? resolvedUriManager;
```

Comment:
make this late and then you can remove all the null asserts where the variable is used later